config/manager/manager.yaml

```
@@ -21,6 +21,7 @@ spec:
         - "--metrics-bind-addr=127.0.0.1:8080"
         - "--leader-elect"
         - "--feature-gates=EKS=${CAPA_EKS:=true},EKSEnableIAM=${CAPA_EKS_IAM:=false},EKSAllowAddRoles=${CAPA_EKS_ADD_ROLES:=false},EKSFargate=${EXP_EKS_FARGATE:=false},MachinePool=${EXP_MACHINE_POOL:=false},EventBridgeInstanceState=${EVENT_BRIDGE_INSTANCE_STATE:=false},AutoControllerIdentityCreator=${AUTO_CONTROLLER_IDENTITY_CREATOR:=true}"
+        - "--v=${CAPA_LOGLEVEL:=4}"
```

```suggestion
        - "--v=${CAPA_LOGLEVEL:=0}"
```

Let's leave the default as 0, and devs can bump it to 4 when they want to.